addons/mail/wizard/mail_resend_message.py

```diff
@@ -34,13 +34,16 @@ def default_get(self, fields):
         if message_id:
             mail_message_id = self.env['mail.message'].browse(message_id)
             notification_ids = mail_message_id.notification_ids.filtered(lambda notif: notif.notification_type == 'email' and notif.notification_status in ('exception', 'bounce'))
-            partner_ids = [Command.create({
-                "partner_id": notif.res_partner_id.id,
-                "name": notif.res_partner_id.name,
-                "email": notif.res_partner_id.email,
+            partner_ids = [({
+                "notification_id": notif.id,
                 "resend": True,
                 "message": notif.format_failure_reason(),
             }) for notif in notification_ids]
+
+            # mail.resend.partner need to exist to be able to execute an action
```

Comment:
This seems highly louche :D . In't it possible to trigger a save if we are still in new mode instead or something like that ?

Reply:
I don't think it's possible, see `action_open_wizard` in `portal_wizard.py`, where I needed to do the exact same thing... (need to create the wizard manually in order to execute button in embeded tree lines)

I can do the same trick instead of creating the partner in the default get, what do you prefer ?

Reply:
I still think it is a technical cost for few added value ... don't really like allowing people to go on notification, which is a completely technical record. Isn't it possible to display more information in this wizard instead ? This one is functional and should help people without sending them to technical records...